functions/cache.test.ts

```diff
@@ -0,0 +1,51 @@
+import { mocked } from '../src/test-utils/mocked'
```

Comment:
This is a collection-specific test. You can cover all of caching with one test file, but then you'd need to cache through a utility function that you could test separately.